app/build.gradle

```diff
@@ -1,10 +1,8 @@
 apply plugin: 'com.android.application'
-
 apply plugin: 'kotlin-android'
-
 apply plugin: 'kotlin-android-extensions'
-
 apply plugin: 'kotlin-kapt'
+apply plugin: 'org.jmailen.kotlinter'
```

Comment:
I'm interested in what are advantages of this plugin.
Maybe you can present us this plugin in demo.

Reply:
Ths is a plugin which gives user an opurtunity to format the kotlin code - i can present it with pleasure.